### ISSUE TITLE
goss: sha256 checksums is relative to a release directory

### ISSUE THREAD

**Issue by deogracia** [aelsabbahy/goss#591]
**Describe the bug**
since 0.3.11, a sha256 file is provided. thanks :D 

For goss, the file is relative to a "release" directory, which I have not.

**How To Reproduce**
* download https://github.com/aelsabbahy/goss/releases/download/v0.3.11/goss-linux-amd64
* download https://github.com/aelsabbahy/goss/releases/download/v0.3.11/goss-linux-amd64.sha256
* run sha256sum -c goss-linux-amd64.sha256 

**Expected Behavior**
```bash
LANG=C sha256sum -c goss-linux-amd64.sha256 
goss-linux-amd64: OK
```

**Actual Behavior**

```bash
LANG=C sha256sum -c goss-linux-amd64.sha256 
sha256sum: release/goss-linux-amd64: No such file or directory
release/goss-linux-amd64: FAILED open or read
sha256sum: WARNING: 1 listed file could not be read
```

**Environment:**
 - Version of goss: 0.3.11
 - OS/Distribution version (if applicable): devuan 2.1 (debian 9 spinoff)
- content of my directory
```bash
ls -a
./  ../  dgoss  .gitkeep  goss-linux-amd64*  goss-linux-amd64.sha256
```

**Comment by aelsabbahy** [comment]
#590 will hopefully fix this.

**Comment by aelsabbahy** [comment]
Fixed by #592
